libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/FeedbackEvent.kt

```diff
@@ -203,22 +184,36 @@ object FeedbackEvent {
      */
     const val CARS_NOT_ALLOWED_ON_STREET = "cars_not_allowed_on_street"
 
-    /**
-     * Feedback description for *routing error*: turn at intersection was unprotected
-     */
-    const val TURN_AT_INTERSECTION_WAS_UNPROTECTED =
-        "turn_at_intersection_was_unprotected"
-
     /**
      * Feedback description for *routing error*: street permanently blocked off
```

Comment:
The category should be *road issue*?

Reply:
road issue exists `line#20` https://github.com/mapbox/mapbox-navigation-android/pull/4794/files#diff-2a926aad48b2853458312661b52f68514e6de8cfa63a1821897c8b04e71426c7R20

Reply:
I mean in the comment. For `const val STREET_PERMANENTLY_BLOCKED_OFF` the comment is `routing error` and for `const val STREET_TEMPORARY_BLOCKED_OFF` it's `road issue`.